src/Tribe/Settings.php

```diff
@@ -284,9 +284,59 @@ public function __construct() {
 		$this->default_tab  = null;
 		$this->current_tab  = null;
 
+		/**
+		 * Once we remove our last usage these internally in Event Tickets and Event Tickets Plus we can
+		 * remove these from our code and keep the magic getter to be able to catch any other usage.
+		 *
+		 * @deprecated 6.1.0
+		 */
+		$this->menuName    = $this->menu_name;
+		$this->requiredCap = $this->required_cap;
+		$this->allTabs     = $this->all_tabs;
+		$this->defaultTab  = $this->default_tab;
+		$this->currentTab  = $this->current_tab;
+		$this->noSaveTabs  = $this->no_save_tabs;
+		$this->adminSlug   = $this->admin_slug;
```

Comment:
Since they are being served by the magic method, why do we set them as well ? If they are set, the magic method won't be triggered and as a result the deprecation notice won't fire. right ?

Reply:
Yeah it's intentional for now until we deprecate the usage of those in ETP and ET. But didn't want to add complexity to the fext release